tests/test_parcels.py

```diff
@@ -486,8 +485,19 @@ def test_tree_buffers_null(self, features_gdf, parcels_gdf):
             calc_buffers=True,
             primary_decision="largest_intersection",
         )
-        # For this test, every result should be NaN for buffers, as there's always a building overlapping the parcel boundary.
-        assert df.filter(like="buffer").head().isna().all().all()
+        # For this test, every parcel has buildings that cross the edges.
```

Comment:
Why does the location of the buildings relative to parcel boundary matter for testing buffers? @mbewley

Reply:
This is just a test. Because nmaipy cannot pull beyond the AOI boundary, this test confirms that buffers are left at "na" if the buffer is invalid (a calculation requires information beyond the AOI).

Reply:
In defensible space usage, it's the user's responsibility to pull AOIs that are large enough.